.github/workflows/ci-rust.yml

```diff
@@ -89,6 +90,22 @@ jobs:
         with:
           tool: nextest@0.9.54, wasm-pack@0.11.0, cargo-deny@0.14.3
 
+      - uses: ./.github/actions/setup-python-poetry
+        id: setup-python
+        with:
+          poetry-version: ${{ env.poetry-version }}
+          project-path: ./server
+        timeout-minutes: 10
+
+      - name: Test cli
+        shell: bash
+        run: |
+          set -ex
+          python make.py python-dev-install
+          cargo build --profile ci-rust --package parsec_cli --features testenv
```

Comment:
Does that step is still required ?

Reply:
Yes, because assert_cmd doesn't build the binary, it uses only

Reply:
Can you do a dedicated step like for the python dependencies then ?